src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -254,8 +256,8 @@ def _write_raw_config(self, conf_block, obj, append=False):
                     "write configuration into rados object %s/%s/%s:\n%s",
                     self.rados_pool, self.rados_namespace, obj, raw_config)
 
-    def _update_common_conf(self, ex_id):
-        common_conf = 'conf-nfs'
+    def _update_common_conf(self, cluster_id, ex_id):
+        common_conf = 'conf-nfs.ganesha-{}'.format(cluster_id)
```

Comment:
same, can we get this from the `ServiceDescription` ?

Reply:
Again in the same doc, it says `
    Orchestrator ServiceDescription rados_config_location should be removed`

Reply:
relates to #34592 According to the discussion I had with @epuertat , we should configure the dashboard using mon commands. using https://docs.ceph.com/ceph-prs/33886/api/mon_command_api/#dashboard-set-ganesha-clusters-rados-pool-namespace etc.

Reply:
Afair we wanted dashboard to call up volume's nfs interface to set-up ganesha clusters. This interface creates a common pool and sets unique namespace for each instance. So why configure dashboard pool and namespace using mon commands?

Reply:
Yes, that's correct. Each cluster has its own set of configs. NFS clusters do not share **anything** except the pool.

Reply:
hrm, the documentation is unlcear, but it does appear that multiple namespaces can specified to the dashboard mon command (using a comma as the delim) --https://github.com/ceph/ceph/blob/8bcfaf29419cb2ddcad8421d7faa9c0a95c6b03b/src/pybind/mgr/dashboard/services/ganesha.py#L35

Reply:
From [doc](https://docs.ceph.com/docs/master/mgr/dashboard/#support-for-multiple-nfs-ganesha-clusters), I see it can be set like this: ```$ ceph dashboard set-ganesha-clusters-rados-pool-namespace <cluster_id>:<pool_name>[/<namespace>](,<cluster_id>:<pool_name>[/<namespace>])*```

It will need to be set every time new cluster is deployed along with old cluster values.